packages/components/src/Components/CullingInfo/CullingInformation.js

```diff
@@ -18,35 +18,37 @@ const calculateTooltip = (culled, warning, currDate) => {
     const warningDate = new Date(warning);
     const diffTime = currDate - warningDate;
     const removeIn = Math.ceil((culledDate - currDate) / days);
+    const msg = `System scheduled for inventory removal in ${removeIn} days`;
     if (diffTime >= 0) {
         return {
             isError: true,
-            msg: `This system is scheduled for removal from inventory in ${removeIn} days (${exact(culledDate)})`
+            msg
         };
     }
 
     return {
         isWarn: true,
-        msg: `This system will be removed from the inventory in ${removeIn} days (${exact(culledDate)})`
+        msg
     };
 };
 
 const CullingInformation = ({ culled, className, staleWarning, stale, currDate, children, render, ...props }) => {
-    if ((new Date(currDate) - new Date(stale)) < 0) {
+    // TODO: remove comments once culling is fine
```

Comment:
What about to remove the comments and let git history keep it?

Reply:
We'd have to go look trough the history, this is easier.